iothub/device/samples/PnpDeviceSamples/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -87,7 +87,7 @@ public async Task PerformOperationsAsync(CancellationToken cancellationToken)
 
             var t = await _deviceClient.GetTwinAsync(s_cancellationToken);
 
-            Properties properties = await _deviceClient.GetPropertiesAsync(s_payloadConvention, s_cancellationToken);
```

Comment:
Re-reading "SubscribeToWritablePropertyEventAsync", I am now reading this a "subscribe to writable-type-of-property event, so property = singular; what do you think?

Reply:
The way I see it is "Subscribe To WritableProperty Event" 

Where we are subscribing to one WritableProperty Event. But since this is the global handler there will always only be one event for all **properties**